apps/site/src/lib/components/Navbar/NavbarHelpers.tsx

```diff
@@ -1,22 +1,25 @@
 import clsx from "clsx";
 import React from "react";
+import Link from "next/link";
 import * as NavMenu from "@radix-ui/react-navigation-menu";
 
 import styles from "./Navbar.module.scss";
 
 const NavLinkItem = React.forwardRef<
 	React.ElementRef<typeof NavMenu.Link>,
 	React.ComponentPropsWithoutRef<typeof NavMenu.Link>
->(({ children, className, ...props }, forwardedRef) => {
+>(({ children, className, href, ...props }, forwardedRef) => {
 	return (
 		<NavMenu.Item>
-			<NavMenu.Link
-				className={clsx(styles.navMenuLink, className)}
-				{...props}
-				ref={forwardedRef}
-			>
-				{children}
-			</NavMenu.Link>
+			<Link href={href as string} passHref legacyBehavior>
```

Comment:
Thought: there may be a way to do this with Radix's `asChild` prop instead of using `legacyBehavior` although I'm not sure if it will actually work.

Reply:
The docs have an example where they use legacyBehavior, so I just adapted that example to here.

I also have no idea how to use the `asChild` prop.